.github/workflows/test.yaml

```diff
@@ -0,0 +1,18 @@
+name: Check Markdown and Examples
```

Comment:
Maybe better name is "Check documentation and STAC examples or we want to run all pre-commit tests here (which we do) then I would just call it "Test"